### PR TITLE
[PM-2331] Update renovate with team groupings

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Configure renovate to tag dependencies with the relevant team.

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -35,14 +36,131 @@
...
-      "matchManagers": ["npm"],
-      "matchUpdateTypes": ["minor", "patch"]
+      "matchPackageNames": ["DnsClient", "Quartz"],
+      "description": "Admin Console owned dependencies",
+      "reviewers": ["team:team-admin-console-dev"]
     },
     {
-      "groupName": "nuget minor",
-      "matchManagers": ["nuget"],
-      "matchUpdateTypes": ["minor", "patch"]
```
Comment (withinfocus) [inline .github/renovate.json:45]: So you also want to get rid of NPM and NuGet grouping?

File: .github/renovate.json [hunk 2]
```diff
@@ -35,14 +36,131 @@
...
+        "Swashbuckle.AspNetCore.SwaggerGen",
+        "xunit",
+        "xunit.runner.visualstudio"
+      ],
+      "description": "Billing owned dependencies",
+      "reviewers": ["team:team-billing-dev"]
+    },
+    {
+      "matchPackageNames": ["CommandDotNet", "dbup-sqlserver", "YamlDotNet"],
+      "description": "Devops owned dependencies",
```
Comment (withinfocus) [inline .github/renovate.json:103]:
```suggestion
      "description": "DevOps owned dependencies",
```

File: .github/renovate.json [hunk 3]
```diff
@@ -35,14 +36,131 @@
...
+        "linq2db.EntityFrameworkCore",
+        "Microsoft.EntityFrameworkCore.Design",
+        "Microsoft.EntityFrameworkCore.InMemory",
+        "Microsoft.EntityFrameworkCore.Relational",
+        "Microsoft.EntityFrameworkCore.Sqlite",
+        "Microsoft.EntityFrameworkCore.SqlServer",
+        "Npgsql.EntityFrameworkCore.PostgreSQL",
+        "Pomelo.EntityFrameworkCore.MySql"
+      ],
+      "description": "Secrets Manager owned dependencies",
```
Comment (withinfocus) [inline .github/renovate.json:127]:
```suggestion
      "description": "SM owned dependencies",
```

Comment (withinfocus) [reply .github/renovate.json:127]: (this is what's in clients)

**Review by withinfocus — CHANGES_REQUESTED** [review]

File: .github/renovate.json (hunk 1 above)
Comment (Hinton) [reply .github/renovate.json:45]: Yes, it makes it difficult to get an overview of the impact.

**Review by withinfocus — APPROVED** [review]

**Review by withinfocus — APPROVED** [review]

**Pull request merged**